.syft.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright The Enterprise Contract Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+---
+exclude:
+  - ./acceptance/**
+  - ./internal/documentation/**
```

Comment:
Not sure if we should be this selective or if it'll matter. Because we feed `/go.mod` to cachi2, it's just going to record all the deps from it anyways. Maybe we just exclude modules completely instead?

Reply:
Huh, not sure, there is SBOM generated by cachi2 and that is based on the configuration, the Syft is instructed to scan the image and source; all of these are merged afterwards and we end up with the SBOM containing the acceptace/tools stuff